packages/firestore/src/platform_node/grpc_connection.ts

```diff
@@ -148,35 +140,79 @@ export class GrpcConnection implements Connection {
     return this.cachedStub.stub;
   }
 
-  invoke(rpcName: string, request: any, token: Token | null): Promise<any> {
+  private getRpc(rpcName: string, token: Token | null): any {
     const stub = this.getStub(token);
+
+    // RPC Methods have the first character lower-cased
+    // (e.g. Listen => listen(), BatchGetDocuments => batchGetDocuments()).
+    const rpcMethod = rpcName.charAt(0).toLowerCase() + rpcName.slice(1);
```

Comment:
Sorry, I wasn't more clear. I meant the mapping technique (a hardcoded map instead of this on-the-fly conversion).

Reply:
I like the on-the-fly conversion since it means we don't have to hardcode all of the RPCs we use.  With WebChannel we can't avoid the hardcoding since the mapping isn't deterministic (whereas I /think/ it is for GRPC).